typescript/typedsql/index.ts

```diff
@@ -0,0 +1,12 @@
+import { PrismaClient } from './node_modules/.prisma/client'
+import { conversionByVariant } from './node_modules/.prisma/client/sql'
```

Comment:
I don't think the `node_modules` prefix is needed.

```suggestion
import { PrismaClient } from `@prisma/client'
import { conversionByVariant } from '.prisma/client/sql'
```